instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components';
+import CommentForm from './CommentForm';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+const IndividualComment = styled.div`
+    margin: 10px 14px;
+    font-size: 1.6rem;
+
+    span {
+        font-weight: bold;
+    }
+`
+
+const Timestamp = styled.p`
+    margin: 14px;
+    font-size: 1.1rem;
+`
+
+
+
+class CommentSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: this.props.comments,
```

Comment:
This is probably an anti-React pattern.  Bolting things onto a child component state object from it's props object can cause issues should probably be avoided.  In addition, in this case you have the data you need on the props object already making this step unnecessary.